addons/binding/org.openhab.binding.smhi/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="smhi"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="weather-and-forecast">
+		<label>SMHI Weather And Forecast</label>
+		<description>Provides current weather and forecast data from the SMHI API.</description>
+
+		<channel-groups>
+			<channel-group typeId="weather" id="current"></channel-group>
+			<channel-group id="forecastTomorrow" typeId="dailyForecast">
+				<label>Tomorrows Forecast</label>
+				<description>This is the weather forecast for tomorrow.</description>
+			</channel-group>
+			<channel-group id="forecastDay2" typeId="dailyForecast">
+				<label>2 Day Forecast</label>
+				<description>This is the weather forecast in two days.</description>
+			</channel-group>
+			<channel-group id="forecastDay3" typeId="dailyForecast">
+				<label>3 Day Forecast</label>
+				<description>This is the weather forecast in three days.</description>
+			</channel-group>
+		</channel-groups>
+
+		<config-description>
+        <parameter name="location" type="text" required="true">

```

Comment:
Formatting looks weird in this file. Can you check here and in your other files and use the automattic formatter (except for the pom.xml). Thanks,